src/components/sidebar.js

```diff
@@ -10,15 +10,15 @@ class Sidebar extends React.Component {
     const md = MarkdownIt();
 
     return (
-      <ul>
+      <ul className="TOC">
         {
           siblings.map((sibling, id) => {
             if (Array.isArray(sibling)) {
               return <li key={id}>{this.renderTransformedToc(sibling, targetLocation)}</li>;
```

Comment:
You should probably also put the `className="TOC-item"` here. It's so `<li>`s with nested `<ul>`s also have the `TOC-item` class applied to the `<li>`

![image](https://cloud.githubusercontent.com/assets/13814048/18973300/bb6d68b8-8651-11e6-8818-45e8a1b06ed4.png)

instead of 

![image](https://cloud.githubusercontent.com/assets/13814048/18973341/dcd9b452-8651-11e6-8ba7-547c5ce6014b.png)